src/ape_test/accounts.py

```diff
@@ -37,6 +37,17 @@ def accounts(self) -> Iterator["TestAccount"]:
     def __len__(self) -> int:
         return len(self._dev_accounts)
 
+    def create_account(self, generation_index: int) -> "TestAccount":
+        generated_account = generate_dev_accounts(
+            self.config["mnemonic"], 1, start_index=len(self) + generation_index
+        )[0]
+        acc = TestAccount(
+            index=len(self) + generation_index,
+            address_str=generated_account.address,
+            private_key=generated_account.private_key,
+        )
+        return acc
+
```

Comment:
Now I'm wondering whether this even needs to be on the container -- this methods just uses `self.config` and `len(self)`, which I assume can be accessed externally. So it would make sense to also move this method to the manager, no? Essentially, this just calls `generate_dev_accounts`.

Reply:
changed this